common/web/types/src/util/util.ts

```diff
@@ -206,6 +207,105 @@ export function describeCodepoint(ch : number) : string {
   return `${s} (U+${Number(ch).toString(16).toUpperCase()})`;
 }
 
+/**
+ * @brief  function to convert a Html-Entity Hex value to a unicode character e.g. &#x63 -> c;  &#x1F60E; -> 😎
+ * @param  hex_Entity the value that will converted
+ * @return a unicode character or undefined if in_str is not recognized
+ */
+export function decodeHexEntity(hex_Entity: string) {
```

Comment:
```suggestion
export function decodeHexEntity(hexEntity: string) {
```

We don't generally use underscores in JavaScript identifiers, except sometimes at the start to indicate a private scope